flow/environment.py

```diff
@@ -225,12 +223,7 @@ def add_args(cls, parser):
             The argument parser where arguments will be added.
 
         """
-        parser.add_argument(
-            "-w",
-            "--walltime",
-            type=float,
-            help="The wallclock time in hours. Deprecated in 0.13 and will be removed in 0.14.",
-        )
+        pass
```

Comment:
Why are we keeping this method?

Reply:
This method is called by all the children of ComputeEnvironment. This has been an empty method for a long time, I just changed it in #476 as a temporary workaround until it was removed in this PR.

Reply:
Got it!